includes/Modules/Idea_Hub.php

```diff
@@ -730,4 +746,37 @@ private function is_idea_post( $post_id ) {
 		return is_array( $this->get_post_idea( $post_id ) );
 	}
 
+	/**
+	 * Adds .googlesitekit-idea-hub__draft class to idea posts on the posts page.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $classes An array of post class names.
+	 * @param array $class An array of additional class names added to the post.
+	 * @param int   $post_id The post ID.
+	 * @return array An array of post class names.
+	 */
+	private function update_post_classes( $classes, $class, $post_id ) {
+		global $post_type;
+
+		$current_screen = get_current_screen();
+		if ( is_null( $current_screen ) || 'edit-post' !== $current_screen->id || 'post' !== $post_type ) {
+			return $notices;
+		}
+
+		if ( $this->is_idea_post( $post_id ) ) {
+			$classes[] = 'googlesitekit-idea-hub__draft';
+
+			if ( ! wp_style_is( 'googlesitekit-admin-css' ) ) {
+				$assets = new Assets( $this->context );
+				// Enqueue fonts.
+				$assets->enqueue_fonts();
+				// Enqueue base admin screen stylesheet.
+				$assets->enqueue_asset( 'googlesitekit-admin-css' );
+			}
```

Comment:
We should avoid this because the class keeps instance properties for ensuring fonts are only enqueued once. Also, `enqueue_asset` would cause all assets to be registered again for a similar reason.

Without using a `Screen` I think our best option here is to fire an action that the main `Assets` instance can respond to.

Lastly, I think it would be a good idea to separate this into two methods – one for just adding the classes, and another that handles the assets.

Reply:
> Without using a Screen I think our best option here is to fire an action that the main Assets instance can respond to.

Added it and updated a few other classes to use the same approach instead of repeating calling `enqueue_fonts()` and `enqueue_asset(...)` methods.

> Lastly, I think it would be a good idea to separate this into two methods – one for just adding the classes, and another that handles the assets.

In this case, these two methods will be 99% identical and will just repeat each other. Not sure whether we really need it. #DRY